frontend/client/components/Proposals/Filters/index.tsx

```diff
@@ -12,16 +12,11 @@ import {
 } from 'api/constants';
 import { typedKeys } from 'utils/ts';
 
-export interface Filters {
-  categories: string[];
-  stage: PROPOSAL_STAGE | null;
-}
-
 interface Props {
   sort: PROPOSAL_SORT;
-  filters: Filters;
+  filters: string[];
   handleChangeSort(sort: PROPOSAL_SORT): void;
-  handleChangeFilters(filters: Filters): void;
+  handleChangeFilters(filters: string[]): void;
```

Comment:
Any reason you didn't want to stick with a structured filters object and do the transformation at the API communication layer? There's a lot of `startsWith` and type casting floating around that looks like it could get hairy.

Reply:
I am lazy. I'll do it.

Reply:
The honesty is appreciated.